cluster/docker/kube-config/kubelet.env

```diff
@@ -0,0 +1,5 @@
+KUBELET_OPTS="--v=2 \
+--address=0.0.0.0 \
+--enable-server \
+--hostname-override=$(hostname -i) "
```

Comment:
What about multiple ip addresses in the output? 
Does it always take the first one or is `awk '{print $1}'` necessary?